.travis.yml

```diff
@@ -64,25 +65,33 @@ addons:
     # https://scan.coverity.com/travis_ci
     project:
       name: "JanusGraph/janusgraph"
-      version: "0.2.0-SNAPSHOT"
```

Comment:
@mbrukman Sorry with that line in there I got a validation warning through https://lint.travis-ci.org/ but then forgot to call it out to you for review. I added #334 to restore it.

I think ideally it'd be great to get the scan to work through the Travis plugin as it seems like that would be the easiest to maintain. But if we can't resolve Coverity build failures then the script will probably be the only option left.